### ISSUE TITLE
We'll need to know your system specs and the specs of the other PC before we can help here.

### ISSUE THREAD

**Issue by Dieter9999** [TheGameCreators/GameGuruRepo#4970]
We'll need to know your system specs and the specs of the other PC before we can help here.

_Originally posted by @davetgc in https://github.com/TheGameCreators/GameGuruRepo/issues/4958#issuecomment-1736794281_

**Comment by Dieter9999** [comment]
Thank you for the reply.

The GGMax PC:

CPU: AMD FX 6350
RAM: 16 GB RAM
OS: Windows 10 Pro
HDD: Seagate Hybrid HDD 2 TB
GPU: NVIDIA GeForce GTX 1060 3 GB

Standalone PC:

Notebook Dell Latitude E7440
CPU: Intel Core i5 4310U
RAM: 8 GB RAM
HDD: 256 GB SSD
GPU: Intel HD

All games made with GameGuru run on this Notebook very vell.

**Comment by davetgc** [comment]
Please add this to the original post, rather than creating a new issue.
